modules/appeals_api/app/controllers/appeals_api/higher_level_reviews/v0/higher_level_reviews_controller.rb

```diff
@@ -86,16 +87,15 @@ def render_model_errors(hlr)
     end
 
     def validate_icn_parameter
```

Comment:
I realized that the errors we were returning for ICN parameter/header validation didn't match our usual `errorModel` schema - raising the exception using `Common::Exceptions::UnprocessableEntity` fixes that